psydac/fem/grid.py

```diff
@@ -127,7 +127,7 @@ def __init__( self, space, start, end, *, quad_order=None, nderiv=1, normalize=F
         # Quadrature data on extended distributed domain
         self._num_elements = ne
         self._num_quad_pts = len( u )
-        self._spans   = np.array( spans   ) - start
+        self._spans   = np.array( spans   )
```

Comment:
yes I have done it for the parallel case, we discussed that at a certain moment , so that the generated code stays the same for serial and parallel case but then i noticed that `example/poisson_2d_mapping` doesn't work anymore so I moved this change to `api/grid.py`

Reply:
I see, thanks!